### ISSUE TITLE
[Bug] [MariaDB] Root limits on MariaDB server 

### ISSUE THREAD

**Issue by click0** [hestiacp/hestiacp#2427]
### Describe the bug

I have mariadb-server-10.6
I'm trying to get a list of users in the database: 

MariaDB [(none)]> select * from mysql.user;
ERROR 1449 (HY000): The user specified as a definer ('mariadb.sys'@'localhost') does not exist

### Tell us how to replicate the bug

1) Open the console on the server
2) Type `mysql`
3) Type `select * from mysql.user;` 

### Which components are affected by this bug?

Database (MariaDB, PostgreSQL)

### Hestia Control Panel Version

1.5.7

### Operating system

Debian 11.2

**Comment by jaapmarcus** [comment]
Looks like an update went wrong or something like that. 

https://jira.mariadb.org/browse/MDEV-22542?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

```
insert into mysql.global_priv values ('localhost','mariadb.sys','{"access":0,"plugin":"mysql_native_password","authentication_string":"","account_locked":true,"password_last_changed":0}');
--
flush privileges;
```

**Comment by click0** [comment]
> Looks like an update went wrong or something like that.

Freshly installed HestiaCP with MariaDB.
No updates have been made.

```
MariaDB [(none)]> insert into mysql.global_priv values ('localhost','mariadb.sys','{"access":0,"plugin":"mysql_native_password","authentication_string":"","account_locked":true,"password_last_changed":0}');
ERROR 1062 (23000): Duplicate entry 'localhost-mariadb.sys' for key 'PRIMARY'
```

**Comment by jaapmarcus** [comment]
mariadb.sys is supposed to be created at install by an installer script provided MariaDB during install of the packages...

**Comment by click0** [comment]
In the case of installing mariadb-server-10.5 (10.5.12), no problems were noticed.

**Comment by jaapmarcus** [comment]
If it was during the same period
https://github.com/hestiacp/hestiacp/issues/2414

It may be cause for it...

**Comment by click0** [comment]
Why did you close the bug report?

**Comment by jaapmarcus** [comment]
Because if there was a  bug in the Hestia installer. Then 10.5 would also not work as it is the same installer. We depend on 3rd parties for thing like this. And it is not a bug in Hestia. 

As you also had an issue with the install of 1.5.7 due to repository issues it would not surprise me that the bug is there some where... 

We have with the 1.5.8 release updated the repository url so there is nothing for us to do ...

**Comment by click0** [comment]
In HestiaCP 1.5.8, only MariaDB 10.5.12 remains working.
The root limits issue remains in MariaDB 10.6.x and 10.7.x branches. 

Whether the [code in HestiaCP](https://github.com/hestiacp/hestiacp/blob/c8161b830128502013807776fa75036c417ec7a9/install/hst-install-debian.sh#L1522-L1532) is at fault further testing will show.

**Comment by click0** [comment]
> In HestiaCP 1.5.8, only MariaDB 10.5.12 remains working. The root limits issue remains in MariaDB 10.6.x and 10.7.x branches.
> 
> Whether the [code in HestiaCP](https://github.com/hestiacp/hestiacp/blob/c8161b830128502013807776fa75036c417ec7a9/install/hst-install-debian.sh#L1522-L1532) is at fault further testing will show.

So it is, I commented out these lines, and the list of users began to be displayed. 
```
mariadb-server-10.6                1:10.6.7+maria~bullseye
```

**Comment by click0** [comment]
1. Command
```
     mysqladmin -u root password $mpass
```
assigns the desired password, but Mariadb can still be accessed using _any_ password, including an empty one. 


2. Command

```
    mysql -e "DELETE FROM mysql.user WHERE password='' AND authentication_string='';"
```
removes the user `mariadb.sys`, which has no password.

**Comment by click0** [comment]
> 1. Command
> 
> 
> ```
>      mysqladmin -u root password $mpass
> ```
> 
> assigns the desired password, but Mariadb can still be accessed using _any_ password, including an empty one.

Instead of https://github.com/hestiacp/hestiacp/blob/c8161b830128502013807776fa75036c417ec7a9/install/hst-install-debian.sh#L1523 use the line: 
```
mysql -e "ALTER USER 'root'@'localhost' IDENTIFIED BY '$mpass'; FLUSH PRIVILEGES;"
```


> 
>     2. Command
> 
> 
> ```
>     mysql -e "DELETE FROM mysql.user WHERE password='' AND authentication_string='';"
> ```
> 
> removes the user `mariadb.sys`, which has no password.

Instead of this delete command, it's better to add an exclusion list. 
```
   mysql -e "DELETE FROM mysql.user WHERE user NOT IN ('mysql.sys', 'mariadb.sys', 'mysqlxsys', 'root') AND password='' AND authentication_string='';"
```
And it's not clear to me what kind of threat you want to get rid of if there is nothing like that in the `/usr/bin/mariadb-secure-installation` script.

**Comment by jaapmarcus** [comment]
Issue has been fixed with #2430
